Density/app/src/main/java/org/cornelldti/density/density/data/MenuClass.kt

```diff
@@ -0,0 +1,31 @@
+package org.cornelldti.density.density.data
+
+class MenuClass(facility: String) {
+    var facilityId: String? = null
+
+    var breakfastItems: ArrayList<CategoryItems>? = null
+    var brunchItems: ArrayList<CategoryItems>? = null
+    var lunchItems: ArrayList<CategoryItems>? = null
+    var liteLunchItems: ArrayList<CategoryItems>? = null
+    var dinnerItems: ArrayList<CategoryItems>? = null
```

Comment:
Avoid using `ArrayList`. Use `List` whenever you can. `List` interface in kotlin is read-only.

Reply:
Also, why is everything mutable and public here?